core/src/test/java/org/frankframework/jdbc/StoredProcedureQuerySenderTest.java

```diff
@@ -90,15 +102,15 @@ public void testSimpleStoredProcedureNoResultNoParameters() throws Exception {
 		assertTrue(result.isSuccess());
 
 		// Due to differences between databases, "rows updated" is sometimes 1, sometimes 0
-		assertTrue("Result should start with [<result><rowsupdated>]", result.getResult().asString().startsWith("<result><rowsupdated>"));
+		assumeTrue(result.getResult().asString().startsWith("<result><rowsupdated>"), "Result should start with [<result><rowsupdated>]");
```

Comment:
Yannick, I didn't check the assume vs assert things before, since I thought you did cover it, since you got remarks on it from Tim. But this one slips through. This `assertTrue` should keep the same name, isn't it?